packages/proposal-relayer/commands/listener.ts

```diff
@@ -0,0 +1,38 @@
+import type { u64 } from "@cennznet/types";
```

Comment:
Nice, maybe we should do this explicitly for all types from now on!?

Reply:
Yeah I'm a fan